Client/src/Components/Inputs/Select/index.jsx

```diff
@@ -0,0 +1,116 @@
+import PropTypes from "prop-types";
+import { useTheme } from "@emotion/react";
+import {
+  MenuItem,
+  Select as MuiSelect,
+  Stack,
+  Typography,
+} from "@mui/material";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+
+import "./index.css";
+
+/**
+ * @component
+ * @param {object} props
+ * @param {string} props.id - The ID attribute for the select element.
+ * @param {string} props.placeholder - The label of the select element.
+ * @param {string} props.placeholder - The placeholder text when no option is selected.
+ * @param {boolean} props.isHidden - Whether the placeholder should be hidden.
+ * @param {string} props.value - The currently selected value.
+ * @param {object[]} props.items - The array of items to populate in the select dropdown.
+ *    @param {string} props.items._id - The unique identifier of each item.
+ *    @param {string} props.items.name - The display name of each item.
+ * @param {function} props.onChange - The function to handle onChange event.
+ * @param {object} props.sx - The custom styles object for MUI Select component.
+ * @returns {JSX.Element}
+ */
+
```

Comment:
Could you add some instructions on how to use this component as well as an example to the JSdocs?

I think pretty much every dev is going to use this componet at some point so some instructions here would be super helpful :+1: